### PR TITLE
[ticket/15083] Fix invalid emoji sizing in ACP & print view

### PR DESCRIPTION
**Pull request by marc1706**
PHPBB3-15083

Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15083

### CODE REVIEW

**Comment by hanakin** [comment]
@marc1706 pls stick to px do not use ems

**Comment by marc1706** [comment]
This is a direct copy from https://github.com/phpbb/phpbb/commit/f111e70fc3959d979e61e8134fcaefb670b32603

Not sure if it's better to hardcode px here (i.e. something like 12px). Suggestions welcome.

**Comment by hanakin** [comment]
its always better to hardcode px over the use of em especially since you never define a font-size on the element. em is a compounded multiple of the font-size of its containing element if not then its parents. It should never be used to define a height or width. 

The entire inclusion method for the img is faulty and needs redone never use img tags instead we need to include it as an svg using the viewbox property to define the aspect and setting the width/height to 100%/auto.

https://css-tricks.com/scale-svg/
http://stackoverflow.com/questions/19484707/how-can-i-make-an-svg-scale-with-its-parent-container

**Comment by hanakin** [comment]
actually the entire inclusion process is faulty svg direct inclusion is recommended over using img tag inclusion for this specific reason

https://demos.emojione.com/latest/sprites-svg.html

File: phpBB/styles/prosilver/theme/print.css [hunk 1]
```diff
@@ -137,3 +137,10 @@ div.spacer { clear: both; }
 .skip_link { display: none; }
 
 .codebox p { display: none; }
+
+.emoji {
```
Comment (hanakin) [inline phpBB/styles/prosilver/theme/print.css]: this needs to be ignored by linting so wrap the class like so pls

/* stylelint-disable declaration-property-unit-whitelist */
.emoji {
	min-height: 18px;
 	min-width: 18px;
 	height: 1em;
 	width: 1em;
 }
/* stylelint-enable declaration-property-unit-whitelist */

**Comment by marc1706** [comment]
Updated

File: phpBB/adm/style/admin.css [hunk 2]
```diff
@@ -2691,3 +2691,10 @@ fieldset.permissions .padding {
 	word-wrap: break-word;
 	word-break: break-all;
 }
+
```
Comment (hanakin) [inline phpBB/adm/style/admin.css]: might as well add the same stylelint comments to here

**Pull request merged**